rtt_ros2_params/src/rtt_ros2_params_service.cpp

```diff
@@ -0,0 +1,86 @@
+// Copyright 2020 Intermodalics BVBA
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#include <string>
+#include <utility>
+
+#include "rtt_ros2_params/rtt_ros2_params.hpp"
+
+#include "rtt/internal/GlobalService.hpp"
+#include "rtt/plugin/ServicePlugin.hpp"
+
+#include "boost/make_shared.hpp"
+
+namespace rtt_ros2_params
+{
+
+bool has_node()
+{
+  return true;
+}
+
+static void loadGlobalROSService()
+{
+  if (!RTT::internal::GlobalService::Instance()->hasService("ros")) {
+    RTT::log(RTT::Error) <<
+      "ROS2 node needs to be loaded before loading ROS2 params" <<
+      RTT::endlog();
+  }
```

Comment:
```suggestion
  RTT::Service::shared_ptr ros =
    RTT::internal::GlobalService::Instance()->provides("ros");
  ros->doc("ROS operations and services");
```

`loadGlobalROSService()` should create the global service `"ros"` if it does not exist yet. `provides()` does that.